server/controllers/aidControllers.js

```diff
@@ -0,0 +1,97 @@
+const db = require("./promise").AidDb;
+import validateAidQueryText from '../validation/aid'
+
+
+const Aids = {
+    async create(req, res) {
+        console.log(req.body);
+        const { errors, isValid } = validateAidQueryText(req.body);
+
+		// Check Validation
+		if (!isValid) {
+			return res.status(400).json(errors);
+		}
+        const queryText = req.body;
+        try {
+          const createdAid = await db.create(queryText);
+          return res.status(201).send({
+            message: 'Aid successfully created',
+            data: createdAid,
+          });
+        } catch (error) {
+          return res.status(400).send(error);
```

Comment:
the status code for errors in a server is `500` not `400` make this change in all your filles